cx_Freeze/freezer.py

```diff
@@ -89,66 +101,28 @@ def __init__(
         self.zipIncludePackages = zipIncludePackages
         self.zipExcludePackages = zipExcludePackages
         self._VerifyConfiguration()
+        return
 
     def _AddVersionResource(self, exe):
-        warning_msg = "*** WARNING *** unable to create version resource"
-        if version_stamp is None:
-            if self.silent < 3:
-                print(warning_msg)
-                print("install pywin32 extensions first")
-            return
-        if not self.metadata.version:
-            if self.silent < 3:
-                print(warning_msg)
-                print("version must be specified")
-            return
-        filename = os.path.join(self.targetdir, exe.target_name)
-        versionInfo = VersionInfo(
-            self.metadata.version,
-            comments=self.metadata.long_description,
-            description=self.metadata.description,
-            company=self.metadata.author,
-            product=self.metadata.name,
-            copyright=exe.copyright,
-            trademarks=exe.trademarks,
-        )
-        version_stamp(filename, versionInfo)
+        return
 
     def _CopyFile(
         self,
         source,
         target,
         copyDependentFiles,
         includeMode=False,
-        machOReference: Optional[MachOReference] = None,
     ):
         if not self._ShouldCopyFile(source):
             return
 
         normalizedSource = os.path.normcase(os.path.normpath(source))
         normalizedTarget = os.path.normcase(os.path.normpath(target))
-        norm_target_name = os.path.basename(normalizedTarget)
 
-        # fix the target path for C runtime files
-        if norm_target_name in self.runtime_files:
-            target_name = os.path.basename(target)
-            target = os.path.join(self.targetdir, "lib", target_name)
-            # vcruntime140.dll must be in the root and in the lib directory
-            if norm_target_name in self.runtime_files_to_dup:
-                self.runtime_files_to_dup.remove(norm_target_name)
-                self._CopyFile(source, target, copyDependentFiles=False)
-                target = os.path.join(self.targetdir, target_name)
-            normalizedTarget = os.path.normcase(os.path.normpath(target))
+        # fix the target path for C runtime files (should do nothing on Darwin...)
+        trarget, normalizedTarget = self._CopyFileNameHook(source=source, target=target, normalizedTarget=normalizedTarget)
```

Comment:
Now have a typo: trarget

Reply:
Sorry, fixed. (That typo would have had no effect, anyway, though, since the function does not do anything in DarwinFreezer.